apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.html

```diff
@@ -280,7 +280,7 @@ <h3 *ngIf="!openDropDown" (dblclick)="openHeaderDropDown()" class="d-inline-bloc
                                 <ng-container *ngIf="txn.type == 'CREDIT'">
 
                                     <td data-title="Date" class="col-xs-2">
-                                        <input class="form-control" autocomplete="off" name="datepic" type="text"
+                                        <input id="giddh-datepicker" class="form-control" autocomplete="off" name="datepic" type="text"
```

Comment:
Change ID, it's Duplicate

Reply:
Done

Reply:
Package-lock JSON added on local but not reflecting on PR